docs/design/automatic-conversation-expiration/automatic-conversation-expiration.md

```
@@ -303,7 +316,73 @@ have to be persisted - it will be recreated after LCore restart.
 | database_reconnection_count | integer | Database reconnection count on startup. When database with conversations is not available on startup, the service tries to reconnect N times with specified delay.           |
 | database_reconnection_delay | integer | Database reconnection delay specified in seconds. When database with conversations is not available on startup, the service tries to reconnect N times with specified delay. |
 
-"2 days 5 hours"
+The `period` is written in a human readable format, for example:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)